iceberg/iceberg-catalog/pom.xml

```diff
@@ -19,6 +19,23 @@
       <groupId>org.apache.hive</groupId>
       <artifactId>hive-iceberg-shading</artifactId>
     </dependency>
+
+    <!-- Intellij dislikes shaded Iceberg: uncomment the following block helps when coding/debugging -->
```

Comment:
cleanup the code

Reply:
Took me a while to find why: 
https://issues.apache.org/jira/browse/HIVE-25027